source/Generic/PlayState/ViewModels/PlayStateManagerViewModel.cs

```diff
@@ -444,6 +460,9 @@ public bool SwitchGameState(PlayStateData gameData, bool isGameStatusOverrided =
                         gameData.IsSuspended = false;
                         notificationType = processesSuspended ? NotificationTypes.Resumed : NotificationTypes.PlaytimeResumed;
                         gameData.Stopwatch.Stop();
+                        gameData.StartPlayingReminderTimer();
+                        gameData.PlayingReminderTimer.Tick += (sender, e) => ShowPlayingReminderNotificationIfGameIsCurrentGame(gameData);
```

Comment:
This subscription will cause issues because there won't be a way to unsubscribe later when disposing the object. Also this being here means that subscriptions will be made each time the method is called, which is not good